dark/score.py

```diff
@@ -13,12 +13,12 @@ class HigherIsBetterScore:
     def __init__(self, score: float) -> None:
         self.score = score
 
-    def __lt__(self, other: "HigherIsBetterScore") -> bool:
+    def __lt__(self, other: object) -> bool:
```

Comment:
Right, that's the intention - to be maximally strict. I think using `object` in this context is the preferred/recommended approach, but I'm not sure. I guess you've already read https://docs.python.org/3/library/typing.html#the-any-type which to me says uses `Any` is basically completely turning off type checking. But seeing as the code here does a strict test, does it make any difference?

Reply:
I think it doesn't make a difference in this case. Using `object` should mean whatever you put in there when you call this method will be accepted. I think it would only make a difference if inside the function you would do something with the variable, i.e. call a function that only accepts a type more specific than `object`.